sensors/baro/lps25xx.c

```diff
@@ -20,7 +20,7 @@
 #include <spi.h>
 #include <sensors-spi.h>
 
-#include "../sensors.h"
+#include <libsensors/sensor.h>
```

Comment:
probably we should update the header guards in `sensor.h` and `client.h` (include `LIBSENSORS` prefix?)

Reply:
Yes, added correct naming in guards.